api/datastore/sql/sql.go

```diff
@@ -508,11 +509,16 @@ func (ds *SQLStore) GetAppByID(ctx context.Context, appID string) (*models.App,
 }
 
 // GetApps retrieves an array of apps according to a specific filter.
-func (ds *SQLStore) GetApps(ctx context.Context, filter *models.AppFilter) ([]*models.App, error) {
-	res := []*models.App{} // for JSON empty list
+func (ds *SQLStore) GetApps(ctx context.Context, filter *models.AppFilter) (*models.AppList, error) {
+	res := &models.AppList{}
```

Comment:
See comment on LHS for (JSON encoding reasons)  the list needs to be initialised as an empty slice or it comes out as `null` in JSON

Reply:
should be a test for this

Reply:
deffo as it doesn't look like that solution actually worked...